docs/site-example/i18n/de.po

```diff
@@ -31,6 +31,13 @@ msgstr ""
 "<%= pubkey %>"
 "</div>"
 
+msgid "gluon-config-mode:novpn"
+msgstr ""
+"<p><div class=\"the-key\">Du hast ausgewählt, <strong>kein Mesh-VPN</strong> "
+"zu nutzen. Dein Knoten kann also nur eine Internetverbindung aufbauen, wenn "
```

Comment:
In config mode it is als labled as "Internetverbindung (Mesh-VPN)", so maybe we should keep it that way here too?